test/e2e/framework/fixture.go

```diff
@@ -282,47 +284,69 @@ func NewSyncerFixture(
 	t *testing.T,
 	resources sets.String,
 	upstream RunningServer,
-	orgClusterName logicalcluster.LogicalCluster,
 	wsClusterName logicalcluster.LogicalCluster,
 ) *SyncerFixture {
-	downstreamServer := NewFakeWorkloadServer(t, upstream, orgClusterName)
-
-	upstreamKcpClusterClient, err := kcpclientset.NewClusterForConfig(upstream.DefaultConfig(t))
+	// Write the upstream logical cluster config to disk for the workspace plugin
+	upstreamRawConfig, err := upstream.RawConfig()
 	require.NoError(t, err)
+	_, kubeconfigPath := writeLogicalClusterConfig(t, upstreamRawConfig, wsClusterName)
 
-	workloadCluster, err := CreateWorkloadCluster(t, upstream.Artifact, upstreamKcpClusterClient.Cluster(wsClusterName), downstreamServer)
-	require.NoError(t, err)
+	// TODO(marun) Configure this with a test arg to support syncer deployment to a pcluster
+	image := "foo"
+
+	// TODO(marun) Maybe support varying the workload cluster name to allow multiple syncers for a given workspace?
```

Comment:
yes please. This is essential very soon.

Reply:
Done.